doc/ImplementationGuide_SmartNIC.md

```diff
@@ -0,0 +1,35 @@
+# Implementation Guide SmartNIC
+
+## Introduction
+This is a guide to implement a generic OpenOffload solution using the gRPC code provided in this repository. 
+
+### Assumptions
+This guide there is a virtual network function (VNF) connected to a SmartNIC. The VNF is responsible for maintaining its own flow table of all sessions, and depending on internal polic, which is invisible to the SmartNIC instructing the SmartNIC to perform actions on the corresponding flows. The initial actions defined in the  gRPC module are:
```

Comment:
Typo:  This guide assumes there is ..
Typo: internal policy,
 Typo: invisible to the SmartNIC, (add comma)

Reply:
Fixed typos